### PR TITLE
feat: port forward and connect to durable GKE networks

### PR DESCRIPTION
**Pull request by smrz2001**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,28 +46,35 @@ Smoke tests live outside of this repo.
...
+So confusingly the `js-ceramic` process exposes the `ComposeDB API` while the `rust-ceramic` process exposes the
+`Ceramic API`.
 
-Eventually we expect the `js-ceramic` repo/process to be renamed to `js-composedb` and the `js-composedb` repo being renamed to `js-compsedb-client` or other similar names.
+Eventually we expect the `js-ceramic` repo/process to be renamed to `js-composedb` and the `js-composedb` repo being
+renamed to `js-compsedb-client` or other similar names.
 
+The `EXTRA_*_URLS` variables are needed because the `*_URLS` are usually injected via `.env` files that are baked into
+the test Docker image, and thus cannot be updated at runtime. The former allow for additional endpoints to be added at
+runtime and used in addition to the formal list of endpoints.
```
Comment (3benbox) [inline README.md:72]: What is runtime in this context, the cerarmic-test?
Are these urls that are generated as part of the test run?
EXTRA_CERAMIC_URLS is unused at this time?

**Review by 3benbox — COMMENTED** [review]
Question about the EXTRA_*_URLS

File: durable/durable-driver.sh [hunk 2]
```diff
@@ -26,11 +26,13 @@ AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID-.}
 AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY-.}
 AWS_REGION=${AWS_REGION-us-east-1}
 
-docker run \
+docker run --network="host" \
```
Comment (nathanielc) [inline durable/durable-driver.sh:29]: We should add a TODO or at least a comment explaining we are using the host network to make portforwarding work. And that we can remove that once we use public URLs in the future

File: README.md (hunk 1 above)
Comment (nathanielc) [reply README.md:72]: Yeah similar question, can we avoid baking the .env files into the image? Maybe not as part of this change

Comment (smrz2001) [reply README.md:72]: Yeah, when the workflow runs, it calls `port-forward.sh`, which will use `kubectl` to port forward into k8s and expose the ComposeDB API locally. Then these `localhost:X` URLs are injected into the tests via the `EXTRA_*_URLS` into the `ceramic-tests` Docker image. These will get concatenated to the existing `*_URLS`.

**Review by 3benbox — APPROVED** [review]
LGTM

**Pull request merged**